apps/artwork_purchase/helpers.coffee

```diff
@@ -0,0 +1,49 @@
+_ = require 'underscore'
+
+module.exports =
+  isEligible: (artwork) ->
```

Comment:
Btw you can request this from the artwork schema in metaphysics now as `is_purchasable`

https://github.com/artsy/metaphysics/pull/484